spring-ai-core/src/main/java/org/springframework/ai/document/Document.java

```diff
@@ -42,6 +42,12 @@ public Document(String id, String text, Map<String, Object> metadata) {
 		this.metadata = metadata;
 	}
 
+	public Document() {
```

Comment:
What necessitates an empty constructor? 
Mind that currently the `Document` class doesn't have setters for the `id`, `text` and `metadata`!